frontend/src/Pages/Inductees.svelte

```diff
@@ -15,6 +17,24 @@
                     return 0;
                 }
             })
+            for (let i = 0; i < inducteesData.length; i++) {
+                if (!majors.includes(inducteesData[i].major)) {
```

Comment:
This way of populating majors is nice, but creates a lot of new fields for what people enter in "Others". Can we try to grab our predefined ones and place all others in "Others"?